pkg/types/baremetal/validation/platform.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/apparentlymart/go-cidr/cidr"
 	"github.com/go-playground/validator/v10"
 	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
```

Comment:
shouldn't this be "github.com/ghodss/yaml" ? it looks like most other yaml imports are using ghodss

Reply:
Thanks @asalkeld it sounds probably a better choice, especially for validation messages as pointed out by @kirankt